test/user.test.js

```diff
@@ -12,7 +12,7 @@ describe('User', function() {
   this.timeout(10000);
 
   var validCredentialsEmail = 'foo@bar.com';
-  var validCredentials = {email: validCredentialsEmail, password: 'bar'};
+  var validCredentials = {email: validCredentialsEmail, password: 'bar', realm: 'foobar'};
```

Comment:
@beeman  would this make previous test using this variable test a different thing, perhaps we should just introduce a new variable to test this feature, then we can ensure the old no realm users work fine too